pkg/grpc/default_api/api_test.go

```diff
@@ -133,6 +139,58 @@ func TestSayHello(t *testing.T) {
 	})
 }
 
+func TestMosnRuntime_publishMessageGRPC(t *testing.T) {
+	t.Run("publish success", func(t *testing.T) {
+		subResp := &runtimev1pb.TopicEventResponse{
+			Status: runtimev1pb.TopicEventResponse_SUCCESS,
+		}
+		// init grpc server
+		mockAppCallbackServer := mock_appcallback.NewMockAppCallbackServer(gomock.NewController(t))
+		mockAppCallbackServer.EXPECT().OnTopicEvent(gomock.Any(), gomock.Any()).Return(subResp, nil)
+
+		lis := bufconn.Listen(1024 * 1024)
+		s := grpc.NewServer()
+		runtimev1pb.RegisterAppCallbackServer(s, mockAppCallbackServer)
+		go func() {
+			s.Serve(lis)
+		}()
+
+		// init callback client
+		callbackClient, err := grpc.DialContext(context.Background(), "bufnet", rawGRPC.WithInsecure(), rawGRPC.WithContextDialer(func(ctx context.Context, s string) (net.Conn, error) {
+			return lis.Dial()
+		}))
+		assert.Nil(t, err)
+
+		cloudEvent := map[string]interface{}{
+			pubsub.IDField:              "id",
+			pubsub.SourceField:          "source",
+			pubsub.DataContentTypeField: "content-type",
+			pubsub.TypeField:            "type",
+			pubsub.SpecVersionField:     "v1.0.0",
+			pubsub.DataBase64Field:      "bGF5b3R0bw==",
+		}
+
+		data, err := json.Marshal(cloudEvent)
+		assert.Nil(t, err)
+
+		msg := &pubsub.NewMessage{
+			Data:     data,
+			Topic:    "layotto",
+			Metadata: make(map[string]string),
+		}
+		a := NewAPI("", nil, nil, nil, nil, nil, nil, nil, nil, nil)
+
+		var apiForTest = a.(*api)
+		//apiForTest.errInt = func(err error, format string, args ...interface{}) {
+		//	log.DefaultLogger.Errorf("[runtime] occurs an error: "+err.Error()+", "+format, args...)
+		//}
+		apiForTest.AppCallbackConn = callbackClient
+		apiForTest.json = jsoniter.ConfigFastest
+		err = apiForTest.publishMessageGRPC(context.Background(), msg)
+		assert.Nil(t, err)
```

Comment:
It seems this test is too simple that it nearly tests nothing. We need to at least test `clientV1.OnTopicEvent` is executed with the right input and output.